udfs/migration/teradata/otranslate.sql

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-CREATE OR REPLACE FUNCTION td.translate(
+CREATE OR REPLACE FUNCTION td.otranslate(
```

Comment:
is this intentional?

Reply:
yes intentional. I'm fixing the name since this function is supposed to be otranslate as the author wrote in README https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs/migration/teradata#otranslatesource_string-string-from_string-string-to_string-string